src/lib.rs

```diff
@@ -24,6 +24,7 @@
 #![deny(unused_imports)]
 #![deny(missing_docs)]
 #![cfg_attr(all(not(test), not(feature = "std")), no_std)]
+#![cfg_attr(docsrs, feature(doc_auto_cfg))]
```

Comment:
Is `doc_auto_cfg` better than `doc_cfg` - I did a quick web search but didn't find an answer. Thanks

Reply:
Oh sweeeeet, `doc_auto_cfg` means no need for `#[cfg_attr(docsrs, doc(cfg(feature = "std")))]`!!!! 

TIL, thanks man.